src/test/kotlin/io/github/rybalkinsd/kohttp/Utils.kt

```diff
@@ -1,13 +1,44 @@
 package io.github.rybalkinsd.kohttp
 
 import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.node.ArrayNode
 import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+
+/**
+ * todo: Probably it would be better to provide extension function to test Json
+ * @author sergey
+ */
+fun assertContainsExactly(expected: Map<String, Any?>, actual: JsonNode) {
+    assertContainsAtLeast(expected, actual)
+
+    actual.fieldNames().forEach {
+        assertTrue(expected.containsKey(it), message = "Expected does not contain $it")
+        val ex = expected[it]
+        if (ex !is List<*>) {
+            assertEquals(ex.toString(), actual[it].asText())
+        }
+    }
+}
 
 /**
  * @author gokul
  */
-fun assertResponses(expected: Map<String, String>, actual: JsonNode) {
-    expected.forEach { (t, u) ->
-        assertEquals(u, actual[t]?.asText())
+fun assertContainsAtLeast(expected: Map<String, Any?>, actual: JsonNode) {
+    expected.forEach { (k, v) ->
+        when (v) {
+            is List<*> -> {
+                val arrayNode = actual[k] as? ArrayNode ?: throw Exception("$k type is not Array, as expected")
+
+                assertEquals(v.size, arrayNode.size())
+
+                arrayNode.forEachIndexed { i, element ->
```

Comment:
Can we leverage `.containsAll` here to check if `actual` has all members of `expected`?